js/bybit.js

```diff
@@ -7511,7 +8260,7 @@ module.exports = class bybit extends Exchange {
         //             ]
         //         },
         //         "retExtInfo": {},
-        //         "time": 1658739027301
+        //         "time": 1676360412576
         //     }
         //
         let fees = this.safeValue (response, 'result', {});
```

Comment:
@sc0Vu can we keep `sign` and `handleErrors` at the bottom for the sake of consistency?